x-pack/plugins/security_solution/public/common/components/events_viewer/index.tsx

```diff
@@ -155,11 +156,19 @@ const StatefulEventsViewerComponent: React.FC<Props> = ({
 
   const globalFilters = useMemo(() => [...filters, ...(pageFilters ?? [])], [filters, pageFilters]);
   const trailingControlColumns: ControlColumnProps[] = EMPTY_CONTROL_COLUMNS;
-  const graphOverlay = useMemo(
-    () =>
-      graphEventId != null && graphEventId.length > 0 ? <GraphOverlay timelineId={id} /> : null,
-    [graphEventId, id]
-  );
+
+  const { DetailsPanel, SessionView, Navigation } = useSessionView({
```

Comment:
This works, not sure if `DetailsPanel` and `SessionView` should be tied together like this, but I would have to think about it more...